admin_guide/manage_nodes.adoc

```diff
@@ -15,115 +15,125 @@ link:../architecture/infrastructure_components/kubernetes_infrastructure.html#no
 in your instance using the link:../cli_reference/overview.html[CLI].
```

Comment:
@adellape This is in the Admin Guide. Do we need the "As an OpenShift Admin.." bit? I believe this is something we've been shying against.